contrib/mkimage/debootstrap

```diff
@@ -28,9 +41,15 @@ rootfs_chroot() {
 # allow for DEBOOTSTRAP=qemu-debootstrap ./mkimage.sh ...
 : ${DEBOOTSTRAP:=debootstrap}
 
+debootstrapPath="$(type -P DEBOOTSTRAP || :)"
```

Comment:
I think you meant `type -P "$DEBOOTSTRAP"` here, right?  Also, the way `$DEBOOTSTRAP` is used below, it technically allows for optional flags to be included, ie: `DEBOOTSTRAP='qemu-debootstrap --something --something'`, although I'm not exactly sure why that'd be necessary.